Make it possible to override version string so we can avoid pick up the
aports git version info.

diff --git a/Makefile b/Makefile
index 0f252d7..bbf2006 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,5 @@
-LDFLAGS += -X main.version=$$(git describe --always --abbrev=40 --dirty)
+VERSION = $$(git describe --always --abbrev=40 --dirty)
+LDFLAGS += -X main.version=$(VERSION)
 
 # default  args for tests
 TEST_ARGS_DEF := -covermode=count -coverprofile=profile.cov
